app/gateway/2.6.x/install-and-run/docker.md

```diff
@@ -220,32 +279,41 @@ The steps involved in starting Kong in [DB-less mode](/gateway/{{page.kong_versi
     ]
     ```
 
-    Notice the `MountPoint` entry. We will need that path in the next step.
+    Notice the `MountPoint` entry. You will need that path in the next step.
+
+### Prepare your configuration file
 
-3. Prepare your declarative configuration file:
+1. Prepare your declarative configuration file in `.yml` or `.json` format.
```

Comment:
Can we add an example declarative config here so that people don't need to think?

```yaml
_format_version: "2.1"
_transform: true

services:
- name: my-service
  url: https://example.com
  routes:
  - name: my-route
    paths:
    - /
```